server/src/main/java/org/elasticsearch/health/HealthService.java

```diff
@@ -35,11 +35,11 @@
 public class HealthService {
 
     // Visible for testing
-    static final String UNKNOWN_RESULT_SUMMARY_PREFLIGHT_FAILED = "Could not determine indicator state. Cluster state is not stable. Check "
-        + "details for critical issues keeping this indicator from running.";
+    static final String UNKNOWN_RESULT_SUMMARY_PREFLIGHT_FAILED = "Could not determine health status. Check details on critical issues "
```

Comment:
We should specify for which indicator we cannot determine the status. 

e.g.: "Could not determine shard allocation health status..."

Reply:
We discussed off-issue and decided to move this to the next edit PR.